addons/arrays/test_shuffle.sqf

```diff
@@ -21,18 +21,14 @@ TEST_OP(count _result,==,count _original,_fn);
     TEST_OP(_x,in,_original,_fn);
 } forEach _result;
 
-// Test depecated version.
+// test modify original
```

Comment:
I'm sorry, but I don't understand, what you want from me.
Could you please be so kind and point it out for me.